README.rst

```diff
@@ -52,7 +45,23 @@ This is optional. You can set your own return url by adding this to `settings.py
 
     PESAPAL_OAUTH_CALLBACK_URL = 'app_name:url_name'  # this needs to be a reversible
 
-#. Run `python manage.py syncdb` to create the polls models.
+The url can have the 'merchant_reference' passed to it as a parameter for further processing.
+Or be without a parameter.
+
+for example the url pattern
+
+r'^response/$'
+
+without the merchant reference parameter
+
+or
+
+r'^response/(?P<merchant_reference>\d+)/$'
```

Comment:
Any particular reason why you'd rather not have this as a query string e.g. `^response/? merchant_reference =xyz`?
It also assumes that merchant string will be an integer